curr_scraper/settings.py

```diff
@@ -69,11 +69,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'DATABASE_URL': os.environ.get('DATABASE_URL'),
```

Comment:
Have you tried running that? I dont remember django supporting this option by default and could not find it in documentation